plugins/irma-client/status-listener.js

```diff
@@ -126,7 +126,7 @@ module.exports = class StatusListener {
   _pollOnce() {
     return (
       // eslint-disable-next-line compat/compat
-      fetch(this._pollingUrl, this._fetchParams)
```

Comment:
Is this cache addition related to the webpack upgrade?

Reply:
I did not explicitly test it, but I'd assume this currently is a problem on `master` too. I cannot imagine `webpack` resolving such an issue.